docs/guides/nrfconnect_factory_data_configuration.md

```diff
@@ -0,0 +1,320 @@
++---0x7200: mcuboot_primary_app (0xf3e00 - 975kB)-+
+| 0x7200: app (0xf3e00 - 975kB)                   |
++-------------------------------------------------+
+| 0xfb000: factory_data (0x1000 - 4kB)            |
+| 0xfc000: settings_storage (0x4000 - 16kB)       |
++-------------------------------------------------+
+
+  sram_primary (0x40000 - 256kB):
++--------------------------------------------+
+| 0x20000000: sram_primary (0x40000 - 256kB) |
++--------------------------------------------+
+
+```
+
+### Generating factory data partition
+
+To store the factory data set in the device's persistent storage, convert the data from the JSON file to its binary representation in the CBOR format.
+To do this, use the [nRF Connect generate partition script](../../scripts/tools/nrfconnect/nrfconnect_generate_partition.py).
+
+To generate the factory data partition using the [nRF Connect generate partition script](../../scripts/tools/nrfconnect/nrfconnect_generate_partition.py), navigate to the *connectedhomeip* root directory and run the following command pattern:
+
+```
+$ python scripts/tools/nrfconnect/nrfconnect_generate_partition.py -i <path_to_JSON_file> -o <path_to_output> --offset <partition_address_in_memory> --size <partition_size>
+```
+In this command:
+
+- ``<path_to_JSON_file>`` is a path to the JSON file containing appropriate factory data.
+- ``<path_to_output>`` is a path to an output file without any prefix. For example, providing ``/build/output`` as an argument will result in creating ``/build/output.hex`` and ``/build/output.bin``.
+- ``<partition_address_in_memory>`` is an address in the device's persistent storage area where a partition data set is to be stored.
+- ``<partition_size>`` is a size of partition in the device's persistent storage area. New data is checked according to this value of the JSON data to see if it fits the size.
+
+To see the optional arguments for the script, use the following command:
+
+```
+$ python scripts/tools/nrfconnect/nrfconnect_generate_partition.py -h
+```
+
+**Example of command for nRF52840 DK:**
+
+```
+$ python scripts/tools/nrfconnect/nrfconnect_generate_partition.py -i build/zephyr/factory_data.json -o build/zephyr/factory_data --offset 0xfb000 --size 0x1000
+```
+
+As a result, ``factory_data.hex`` and ``factory_data.bin`` files are created in the ``/build/zephyr/`` directory. The first file contains the memory offset.
+For this reason, it can be programmed directly to the device using a programmer (for example, nrfjprog).
+
+
+<hr>
+<a name="Building an example with factory data"></a>
+
+## Building an example with factory data
+
+You can generate the factory data set manually using the instructions described in the [Generating factory data](#generating-factory-data) section.
+Another way is to use the nRF Connect build system that generates factory data content automatically using Kconfig options and includes the content in the final firmware binary.
```

Comment:
Do you mean nRF Connect *platform* build system here or nRF Connect *SDK* one?

Reply:
I mean the nRF Connect platform build system because a script that creates factory data and the nRF Connect SDK build system are parts of the Matter build system. So I'll add a *platform* word to this sentence.